docs/attribute_analysis/drafts/license_analysis.md

```diff
@@ -154,10 +552,122 @@ License metadata is not only expressed in different formats, but also stored in
+- **License expression support**: No native support for SPDX expressions. Only individual SPDX identifiers can be specified, either as a single value in `license` or as multiple values in the `licenses` array. Complex licensing scenarios (AND, WITH, exceptions) cannot be expressed using the built-in metadata fields.
+- **Location**: Declared in the `.gemspec` file, which is written in Ruby and located at the gem's root directory. When a gem is packaged and distributed, the `.gemspec` metadata is embedded within the `.gem` file (which is a tar archive). License files (typically named `LICENSE`, `LICENSE.txt`, or similar) are expected to be included in the gem package but this is not enforced. The metadata is extractable from the `.gem` file and is also served by RubyGems.org through its web interface and API.
+- **Notes**: The `license` and `licenses` fields are optional, and no validation is performed on their content. Because the `.gemspec` is a Ruby script, it must be executed to extract metadata, which can pose security considerations. The lack of enforced standards means license data quality varies significantly across gems.
+
+### macOS Ecosystem — Homebrew
+- **Data type**: Formula files are Ruby scripts (DSL) that define package specifications. The `license` method accepts: SPDX identifier strings (e.g., `"MIT"`, `"Apache-2.0"`), special symbols (`:public_domain`, `:cannot_represent`), arrays for multiple licenses, hashes with `any_of:` or `all_of:` keys for complex expressions, and hash syntax for license exceptions (`"License" => { with: "Exception" }`).
+- **License expression support**: Full native support for SPDX license expressions through structured Ruby data types based on the SPDX License Expression Guidelines. The `any_of:` hash key represents OR relationships, `all_of:` represents AND relationships, and the hash syntax `"License" => { with: "Exception" }` represents SPDX WITH operators for license exceptions (e.g., `"Apache-2.0" => { with: "LLVM-exception" }`). These structures can be arbitrarily nested to express any valid SPDX license expression. Version operators (`+`, `-only`, `-or-later`) are supported directly in license identifier strings.
+- **Location**: Declared in Formula files (`.rb` Ruby scripts) located in Homebrew's formula repositories (homebrew-core, homebrew-cask, and third-party taps). The formula metadata is not embedded in installed packages but is maintained in Git repositories. Homebrew's API and web interface (formulae.brew.sh) serve this metadata. License information is stored in Homebrew's formula repositories and synced to the local system when formulae are updated.
+- **Notes**: The `license` field is required for new formulae in homebrew-core. Formula files must be executed as Ruby code to extract metadata. Homebrew uses standard SPDX identifiers in string format, not Ruby symbol notation.
```

Comment:
"Homebrew uses standard SPDX identifiers in string format, not Ruby symbol notation." is that right?